tests/extensions/default/analyzers/Microsoft.DotNet.UpgradeAssistant.Extensions.Default.Analyzers.Test/Microsoft.DotNet.UpgradeAssistant.Extensions.Default.Analyzers.Test.csproj

```diff
@@ -14,6 +14,9 @@
     <Content Include="assets\**">
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </Content>
+    <None Include="..\..\..\..\..\src\extensions\default\analyzers\Microsoft.DotNet.UpgradeAssistant.Extensions.Default.Analyzers.Package\build\WebTypeReplacements.typemap" Link="WebTypeReplacements.typemap">
```

Comment:
If WebTypeReplacements.typemap lived in the analyzers project and the analyzer package project linked it from there, then it would show up in projects like this which depend on the analyzers automatically. Linking is ok, but it's nice to keep it to a minimum since it adds file path-specific dependencies outside of the current project.

Reply:
Agreed. The reason it was done like this is so the webtypereplacements.typemap doesn't show up in the default extension.